config/routes.rb

```diff
@@ -348,7 +348,7 @@
 
   # You can have the root of your site routed with "root
   get "/robots.:format" => "pages#robots"
-  get "/api", to: redirect("https://docs.dev.to/api")
+  get "/api", to: redirect("https://docs.forem.com/api")
```

Comment:
Not markdown, but found this link as well while combing the code base for docs.dev.to.